aptos-move/framework/aptos-stdlib/sources/cryptography/ristretto255_bulletproofs.move

```diff
@@ -195,7 +347,63 @@ module aptos_std::ristretto255_bulletproofs {
     }
 
     #[test(fx = @std)]
-    #[expected_failure(abort_code = 0x010001, location = Self)]
+    fun test_batch_prover(fx: signer) {
+        features::change_feature_flags_for_testing(&fx, vector[ features::get_bulletproofs_batch_feature() ], vector[]);
+
+        let vs = vector[
+            ristretto255::new_scalar_from_u64(59),
+            ristretto255::new_scalar_from_u64(60),
+        ];
+        let rs = vector[
+            std::option::extract(&mut ristretto255::new_scalar_from_bytes(A_BLINDER)),
+            std::option::extract(&mut ristretto255::new_scalar_from_bytes(A_BLINDER)),
```

Comment:
Did you mean `B_BLINDER` here?

Reply:
(In other words, why re-use the same blinder? It's typically not a safe thing to do, unless care is exercised.)

Reply:
Done